MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -386,32 +387,33 @@ private static TranscodeReason GetTranscodeReasonForFailedCondition(ProfileCondi
         /// <param name="type">The <see cref="DlnaProfileType"/>.</param>
         /// <param name="playProfile">The <see cref="DirectPlayProfile"/> object to get the video stream from.</param>
         /// <returns>The normalized input container.</returns>
-        public static string? NormalizeMediaSourceFormatIntoSingleContainer(string inputContainer, DeviceProfile? profile, DlnaProfileType type, DirectPlayProfile? playProfile = null)
+        public static string NormalizeMediaSourceFormatIntoSingleContainer(string inputContainer, DeviceProfile? profile, DlnaProfileType type, DirectPlayProfile? playProfile = null)
         {
-            if (string.IsNullOrEmpty(inputContainer))
+            if (!inputContainer.Contains(',', StringComparison.OrdinalIgnoreCase) || profile is null)
```

Comment:
I prefer putting the cheapest operation first (in this case we don't have to de the relatively expensive contains operation if profile is null)
```suggestion
            if (profile is null || !inputContainer.Contains(',', StringComparison.OrdinalIgnoreCase))
```